src/routes/users.js

```diff
@@ -0,0 +1,12 @@
+import express from 'express';
+import UsersController from '../controllers/usersController';
+import authMiddleware from '../middlewares/auth';
+import { profileSchema } from '../validation/schemas';
+import validate from '../validation/validator';
+
+const router = express.Router();
+
+router.get('/profile', authMiddleware, UsersController.getProfile);
+router.patch('/profile', authMiddleware, validate(profileSchema, 'body'), UsersController.updateProfile);
+
+export default router;
```

Comment:
Please move this also to the Auth route file, don't create yours.

Reply:
Alright, noted